packages/core/src/components/non-ideal-state/nonIdealState.tsx

```diff
@@ -14,22 +14,22 @@ import { IProps } from "../../common/props";
 
 export interface INonIdealStateProps extends IProps {
     /*
-     * An action the user can take to correct the non-ideal state.
+     * An action that's attached to the non-ideal state.
```

Comment:
hmm not a fan of this new wording. the previous wording expressed the intent of the prop.